tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -960,7 +960,7 @@ def _profiler_config(
                         "quantile_statistic_interpolation_method": "nearest",
                         "quantile_bias_correction": False,
                         "quantile_bias_std_error_ratio_threshold": None,
-                        "estimator": "bootstrap",
+                        "estimator": "exact",
```

Comment:
@NathanFarmer Per earlier comment -- let us leave `bootstrap` as the variable value.

Reply:
✅ after our offline conversation